static/src/stylesheets/module/commercial/paidfor/_paidfor-content.scss

```diff
@@ -57,10 +57,12 @@
     }
 }
 
-.paidfor-box {
+.paidfor-box--header,
```

Comment:
Having `.paidfor__box.paidfor__box--text` and `.paidfor-box.paidfor-box--text` are almost the same think I think. `.paidfor-box` could definitely be it's own block element.

I think my concern here is that having a modifier only (which is what these DOM elements have) moves away from BEM completely (a modifier should be used to modify and existing block).

In terms of the dropdown - that makes sense too... Is it abstracted enough to be reused?